app/uk/gov/hmrc/plasticpackagingtax/registration/views/members_under_group_control_page.scala.html

```diff
@@ -0,0 +1,63 @@
+@*
+ * Copyright 2021 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *@
+
+@import uk.gov.hmrc.plasticpackagingtax.registration.controllers.{routes => pptRoutes}
+@import uk.gov.hmrc.plasticpackagingtax.registration.views.html.components._
+@import uk.gov.hmrc.plasticpackagingtax.registration.views.html.main_template
+@import uk.gov.hmrc.plasticpackagingtax.registration.views.model.{BackButton, Title}
+@import uk.gov.hmrc.plasticpackagingtax.registration.forms.MembersUnderGroupControl
+
+@this(
+  formHelper: FormWithCSRF,
+  govukLayout: main_template,
+  govukFieldset: GovukFieldset,
+  govukRadios : GovukRadios,
+  saveButtons: saveButtons,
+  errorSummary: errorSummary
+)
+
+
+@(form: Form[MembersUnderGroupControl])(implicit request: Request[_], messages: Messages)
+
+@govukLayout(
+  title = Title("group.membersUnderGroupControl.title"),
+  backButton = Some(BackButton(messages("site.back"), pptRoutes.RegistrationController.displayPage(), messages("site.back.hiddenText")))) {
```

Comment:
Should the back link be to the `RegistrationTypeController`?

Reply:
Done